eessi/testsuite/eessi_mixin.py

```diff
@@ -120,9 +120,6 @@ def measure_mem_usage(self):
     def set_tag_ci(self):
         "Set CI tag if bench_name_ci and bench_name are set and are equal"
         if self.bench_name_ci:
-            if not self.bench_name:
```

Comment:
I'm confused about what breaks in your QE test if you keep this if-statement? I mean: you set `self.bench_name` in the QE test after the `init` step, but since this hook in `eessi_mixin` is `always_last`, by that time, both `self.bench_name` and `self.bench_name_ci` have been set, no?

Reply:
Never mind, I see `self.bench_name` is only set for the `CI` case. The others have no `self.bench_name` set.